backend/src/utils/notebookUtils.ts

```diff
@@ -261,6 +261,7 @@ export const assembleNotebook = async (
         'opendatahub.io/dashboard': 'true',
       },
       annotations: {
+        'image.openshift.io/triggers': `'[{"from":{"kind":"ImageStreamTag","name":"${imageSelection}", "namespace":"${namespace}"},"fieldPath":"spec.template.spec.containers[?(@.name==\"${name}\")].image"}]'`, 
```

Comment:
ah, yeah, since I have the backticks there, the single quotes are not needed in the value. The backslash as escape character is definitely needed, though.